eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -109,9 +110,20 @@ public Resource getCoreInfo(@PathVariable("variantCoreString") String variantCor
     }
 
     private void checkParameters(String variantCoreString, String annotationVepVersion,
-                                 String annotationVepCacheVersion, String species, String assembly)  {
+                                 String annotationVepCacheVersion, String species, String assembly) {
         if (!variantCoreString.contains(":")) {
             throw new IllegalArgumentException("Please describe a variant as 'sequence:location:reference:alternate'");
+        } else {
```

Comment:
This doesn't need to be nested under an `else` branch, as throwing an exception will interrupt the method execution. In fact, I think both checks are a bit redundant, because the split + length check effectively covers the contains check too.

Reply:
Fixed :)